hillel-expense-tracker-dao/src/main/java/com/imaks/App.java

```diff
@@ -0,0 +1,13 @@
+package com.imaks;
+
+/**
+ * Hello world!
+ *
+ */
+public class App 
```

Comment:
no need to create main class. Just put the package-info file, so that git doesn't ignore it